app/controllers/logs_searches_controller.rb

```diff
@@ -1,6 +1,7 @@
 class LogsSearchesController < ApplicationController
   def create
     @search = LogsSearch.new(search_params)
+    @locations = current_organisation.locations.order([:address])
```

Comment:
how come this isn't in the `location` controller?

Reply:
Because we are using the LogsSearch controller to retrieve the logs for each filter

Reply:
I do feel this LogsSearchController is a tad confusing imo

Reply:
Looks like its in the wrong controller action but doesn't work the other way. I'm also a little confused about how this controller is setup and how the routes work.